ydb/tests/library/harness/daemon.py

```diff
@@ -253,10 +252,10 @@ def copy_file_or_dir(self, file_or_dir, target_path):
 
         transit_path = os.path.join(self._artifacts_path, os.path.basename(file_or_dir))
 
-        self.ssh_command(['sudo', 'rm', '-fr', target_path], raise_on_error=True)
+        self.ssh_command(['sudo', 'rm', '-rf', target_path], raise_on_error=True)
 
         self._run_in_subprocess(
-            ["scp", '-r', file_or_dir, self._path_at_host(transit_path)],
+            ["scp"] + self._ssh_options + [ '-r', file_or_dir, self._path_at_host(transit_path)],
```

Comment:
Without that options there was prompt from scp (`Are you sure you want to continue connecting ...`) and the test process was hung.